### PR TITLE
use button as form submit element

### PR DESCRIPTION
**Pull request by buckrivard**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -2,17 +2,20 @@ import Image from "next/image";
...
 import Layout from "../components/layout";
 import SubscribeForm from "../components/SubscribeForm";
-import Head from "next/head";
 import Link from "next/link";
 import { useEffect, useState } from "react";
 
+const HIDE_SUBSCRIBER_MODAL_KEY = "hide-subscription-modal";
+
+const getIsModalHidden = () => localStorage.getItem(HIDE_SUBSCRIBER_MODAL_KEY) === "true";
+const hideModal = () => localStorage.setItem(HIDE_SUBSCRIBER_MODAL_KEY, "true");
```
[the remark this answers is not in the document]
Comment (buckrivard) [reply pages/index.js:11]: nope, this is to set the flag for 'hidden' state to true

**Review by joeday — CHANGES_REQUESTED** [review]
@buckrivard a few questions:

1. The modal is not revealing itself. I tested on multiple browsers in private windows. 
2. Can you update the `SubscribeForm.module.css` file so that the styles applied to the `input[type="submit"]` applies to the button?

**Comment by joeday** [comment]
This is looking good to me. If you're done, I'm ready to merge it.

**Review by joeday — APPROVED** [review]
Looks good. Let's goooo

**Pull request merged**